packages/SwingSet/test/bootstrap-relay.js

```diff
@@ -46,18 +54,48 @@ export const buildRootObject = () => {
     },
 
     /**
-     * Turns an object into a remotable by ensuring that each property is a function
+     * Derives a remotable from an object by mapping each object property into a method that returns the value.
+     * This function is intended to be called from testing code outside of the vat environment,
+     * but methods of the returned remotable are intended to be called from *inside* that environment
+     * and therefore do not perform argument/response translation.
      *
      * @param {string} label
      * @param {Record<string, any>} methodReturnValues
      */
-    makeSimpleRemotable: (label, methodReturnValues) =>
-      // braces to unharden so it can be hardened
-      encodePassable(
-        Far(label, {
-          ...objectMap(methodReturnValues, v => () => decodePassable(v)),
-        }),
-      ),
+    makeRemotable: (label, methodReturnValues) => {
+      const callLogs = [];
+      const makeGetterFunction = (value, name) => {
+        const getValue = (...args) => {
+          callLogs.push([name, ...args]);
+          return value;
+        };
+        return getValue;
+      };
+      // `objectMap` hardens its result, but...
+      const methods = objectMap(methodReturnValues, (encodedValue, key) => {
+        const ret = decodePassable(encodedValue);
+        return makeGetterFunction(ret, key);
+      });
```

Comment:
I think this can be simplified even further.
```suggestion
    makeRemotable: (label, encodedMethodReturnValues) => {
      const callLogs = [];
      const makeGetterFunction = (value, name) => {
        const getValue = (...args) => {
          callLogs.push([name, ...args]);
          return value;
        };
        return getValue;
      };
      const methodReturnValues = decodePassable(encodedMethodReturnValues);
      // `objectMap` hardens its result, but...
      const methods = objectMap(methodReturnValues, makeGetterFunction);
```